copy/copy.go

```diff
@@ -195,6 +194,10 @@ func Image(ctx context.Context, policyContext *signature.PolicyContext, destRef,
 	} else {
 		c.compressionFormat = *options.DestinationCtx.CompressionFormat
 	}
+	if options.DestinationCtx == nil {
```

Comment:
Shouldn’t this have been `!=`?

Reply:
:man_facepalming: yes ... doing work in a hurry has it's consequences. I'll add a fix-fix-fix.

Reply:
Thanks for checking, @mtrmac! PR is here https://github.com/containers/image/pull/690. Apologies for the sloppiness.